[Bug, lake] table_bronze_pdr_slots.py : line-too-long

```text
(venv) trentmc@trents-m3: ~/code/pdr-backend $ pylint pdr_backend/*
************* Module table_bronze_pdr_slots
pdr_backend/lake/table_bronze_pdr_slots.py:88:0: C0301: Line too long (107/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:97:0: C0301: Line too long (165/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:99:0: C0301: Line too long (149/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:108:0: C0301: Line too long (119/100) (line-too-long)
```

Just entered main branch in the last hour

> This is not actually an issue, if running mypy properly:

It was actually pylint

And I'm still getting the issue

```text
(venv) trentmc@trents-m3: ~/code/pdr-backend $ pylint --rcfile .pylintrc * pdr_backend/*
************* Module table_bronze_pdr_slots
pdr_backend/lake/table_bronze_pdr_slots.py:88:0: C0301: Line too long (107/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:97:0: C0301: Line too long (165/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:99:0: C0301: Line too long (149/100) (line-too-long)
pdr_backend/lake/table_bronze_pdr_slots.py:108:0: C0301: Line too long (119/100) (line-too-long)
```

--- a/pdr_backend/lake/table_bronze_pdr_slots.py
+++ b/pdr_backend/lake/table_bronze_pdr_slots.py
@@ -1,3 +1,4 @@
+# pylint: disable=line-too-long
 import logging
 from collections import OrderedDict
 from typing import Callable

